### PR TITLE
Modified to configure web security for webkit 2.38.

### PR DESCRIPTION
**Pull request by neeraj-deshpande**

### CODE REVIEW

File: WebKitBrowser/WebKitImplementation.cpp [hunk 1]
```diff
@@ -2231,10 +2231,15 @@ static GSourceFuncs _handlerIntervention =
 
             // Allow mixed content.
             bool enableWebSecurity = _config.Secure.Value();
+#if WEBKIT_CHECK_VERSION(2, 38, 0)
```
Comment (modeveci) [inline WebKitBrowser/WebKitImplementation.cpp:2234]: Dont you think it is better to check here as "greater than" 2.38 iso "equal"?

File: WebKitBrowser/WebKitImplementation.cpp [hunk 2]
```diff
@@ -2231,10 +2231,15 @@ static GSourceFuncs _handlerIntervention =
...
+            g_object_set(G_OBJECT(preferences),
+                     "disable-web-security", !enableWebSecurity, nullptr);
+#else
             g_object_set(G_OBJECT(preferences),
                      "enable-websecurity", enableWebSecurity,
                      "allow-running-of-insecure-content", !enableWebSecurity,
```
Comment (modeveci) [inline WebKitBrowser/WebKitImplementation.cpp:2240]: I am checking with Miguel, what happened to these properties, I will update you

File: WebKitBrowser/WebKitImplementation.cpp (hunk 1 above)
Comment (modeveci) [reply WebKitBrowser/WebKitImplementation.cpp:2234]: Correction: 
I mean >= 2.38

Comment (neeraj-deshpande) [reply WebKitBrowser/WebKitImplementation.cpp:2234]: It checks for >= 2.38.

https://github.com/WebPlatformForEmbedded/WPEWebKit/blob/362d5b37f9f72286b906c40e2785d351dcce6eac/Source/WebKit/UIProcess/API/wpe/WebKitVersion.h.in#L67

**Review by modeveci — APPROVED** [review]

**Review by MFransen69 — APPROVED** [review]

**Comment by modeveci** [comment]
This PR came to master branch by syncing with rdkservices. Closing

**Pull request closed without merging**